### ISSUE TITLE
Is it correct that the avg loss is high? Or is there something to be adjusted

### ISSUE THREAD

**Issue by willyman870321** [AlexeyAB/darknet#8556]
Is it correct that the avg loss is high? 
mAP is not bad
Or is there something to be adjusted
![loss](https://user-images.githubusercontent.com/50653106/173476953-c39599de-28c2-487b-89a8-959a9461c10d.png)

**Comment by stephanecharette** [comment]
> Is it correct that the avg loss is high?

No, normally you want your loss to be < 1.0.  I'd say something isn't quite right.

**Comment by albertfaromatics** [comment]
- Which model is it?
- How many samples do you have?
- How many classes do you have?

More information is needed to know if this high loss is normal. All models will have different loss, you should not believe or compare with other people

**Comment by willyman870321** [comment]
I have all 15000 photos
23 classes 

1. 2304 photos
2. 4243 photos
3. 2507 photos
4. 373 photos
5. 1109 photos
6. 6978 photos
7. 943 photos
8. 2630 photos
9. 6145 photos
10. 3079 photos
11. 4802 photos
12. 7570 photos
13. 676 photos
14. 328 photos
15. 9289 photos
16. 5228 photos
17. 2301 photos
18. 2252 photos
19. 455 photos
20. 342 photos
21. 788 photos
22. 8388 photos
23. 1388 photos

Above is the number of photos for each category
Could it be data imbalance?

**Comment by albertfaromatics** [comment]
It could, yes. Also, how many boxes per image do you have? The more boxes per image, the higher the loss will be.

I'd just try to test it with new images and see the results. Usually loss is "not that important" as it is just a parameter to minimize. Some times the minimum value can be "high".

Could you check while training the individual losses? (class, iou, etc) They are printed per minibatch

**Comment by willyman870321** [comment]
Each picture has 7-8 boxes and they are all different classes

**Comment by Soplia** [comment]
I think you didn't enable 'mosaic' in your cfg file.
